src/node.d.ts

```diff
@@ -0,0 +1,10 @@
+import { CloseEvent, Event, ErrorEvent, MessageEvent } from 'ws';
+import { ReliableWS as WS } from './index.js';
+
+export declare class ReliableWS extends WS {
+  onopen: ((event: Event) => void) | null;
+    onerror: ((event: ErrorEvent) => void) | null;
+    onclose: ((event: CloseEvent) => void) | null;
+    onmessage: ((event: MessageEvent) => void) | null;
+
+}
```

Comment:
@bpanthi977 Aren't we able to get rid of this file now ?

Reply:
Yes we are. 
Oversight from my part.